src/python/requirements.txt

```diff
@@ -22,3 +22,5 @@ PyJWT==1.5.0
 # service
 python-daemon==2.1.2
 
+# process
+psutil==5.2.2
```

Comment:
there are other way too, issue command, read files/ output and parse etc..., but that will take time.